src/pages/Admin/Admin_v2.tsx

```diff
@@ -97,61 +53,16 @@ const Admin_v2: React.FC = () => {
     return <>Forbidden</>;
   }
 
-  const renderVenueView = () => {
-    switch (selectedOption) {
-      case SidebarOptions.dashboard:
-        return (
-          <VenueDetails
-            venue={selectedVenue as Venue_v2}
-            onSave={() => setSelectedOption(sidebarOptions[0].id)}
-          />
-        ); // Venue_v2 is incomplete with typing (lags behind latest Venue)
-
-      case SidebarOptions.basicInfo:
-        return (
-          <BasicInfo
-            venue={selectedVenue as Venue_v2}
-            onSave={() => setSelectedOption(sidebarOptions[0].id)}
-          />
-        );
-
-      case SidebarOptions.entranceExperience:
-        return (
-          <EntranceExperience
-            venue={selectedVenue as Venue_v2}
-            onSave={() => setSelectedOption(sidebarOptions[0].id)}
-          />
-        );
-
-      case SidebarOptions.advancedMapSettings:
-        return (
-          <AdvancedSettings
-            venue={selectedVenue as Venue_v2}
-            onSave={() => setSelectedOption(sidebarOptions[0].id)}
-          />
-        );
-
-      // case SidebarOptions.ticketingAndAccess:
-      //   return <TicketingAndAccess />;
-
-      default:
-        return null;
-    }
-  };
-
   return (
     <>
-      <S.Wrapper hasSelectedVenue={!!selectedVenue}>
-        {selectedVenue && (
-          <AdminSidebar
-            sidebarOptions={sidebarOptions}
-            selected={selectedOption}
-            onClick={setSelectedOption}
-          />
-        )}
-
+      <S.Wrapper className="no-venue-selected">
         <S.ViewWrapper>
-          {selectedVenue ? renderVenueView() : <AdminVenues venues={venues} />}
+          {selectedVenue ? (
+            // @debt Venue_v2 has different structure than AnyVenue, 1 of them should be deprecated.
+            <AdminVenueView venue={selectedVenue as Venue_v2} />
```

Comment:
👍

Reply:
Is that something that can be handled in this PR (or an upcoming followup)?

Reply:
I think it's an OOS task